frontend/src/components/Discover_Event/DiscoverEvent.tsx

```diff
@@ -1,28 +1,22 @@
 import React from "react";
-import HeroSection from "./HeroSection";
-import NavigationButtons from "./NavigationButtons";
-import EventList from "./EventList";
-import FilterSection from "./FilterSection";
+import EventList from "./event-list/EventList";
+import FilterSection from "./filter-section/FilterSection";
 import Pagination from "./Pagination";
 
-const App: React.FC = () => {
+const DiscoverEvent: React.FC = () => {
   return (
     <div>
-      <HeroSection />
-      <NavigationButtons />
-      <div className="grid grid-cols-2 gap-4">
-        <div>
-          <FilterSection />
-        </div>
-        <div>
+      <div className="grid grid-cols-6 ps-8 pt-8">
+        <FilterSection />
+        <div className="col-span-4">
           <EventList displayMode="list" />
         </div>
       </div>
-      <div className=" inset-x-0 bottom-0 flex justify-center pb-4 w-full ">
+      <div className="flex justify-center pb-4">
```

Comment:
Can remove the div here and in the Pagination Component used another name such as PaginationComponent, then you can add the pagination component from shadCN, refer to screenshot below, notice line 14 and line 38. The reason is the pagination from shadcn already has property flex and justify-center

![Screenshot 2024-05-04 at 11 11 39 PM](https://github.com/Thangnguyenvn98/GatheringGlobe/assets/69278249/8864865f-45dd-44fd-993f-b78b1f0e3610)